redhat-cop/infra-ansible
# Add MTU Network config to support Jumbo Frames

### What does this PR do?
Adds MTU inventory configuration to network devices

### How should this be tested?
Automated tests are preferred, but not always doable - especially for infrastructure. Include commands to run your new feature, and also post-run commands to validate that it worked.

Manual test required to verify the interface has the proper MTU set and configured.
`ip addr` will show the setting but network services will need to be restarted to take effect.


### People to notify
cc: @oybed
